substrate/frame/support/procedural/src/construct_runtime/expand/inherent.rs

```diff
@@ -134,41 +138,23 @@ pub fn expand_outer_inherent(
 											return result;
 										}
 									}
+									continue
```

Comment:
Maybe we shouldn't do `continue` here. A pallet can declare a check for inherent for a call that is not its own call.
It was actually the case for pallet-babe a long time ago, which was doing some check inherent on the pallet-timestamp call.
Hopefully nobody do this anymore but still.

(Also inherent is IMO an invalid concept for parachains given its security is not inherited from the relay chain. but this is another story.)